components/breadcrumb/index.css

```diff
@@ -1,333 +1,261 @@
+	--spectrum-breadcrumbs-action-button-color: var(--spectrum-neutral-subdued-content-color-default);
+	--spectrum-breadcrumbs-action-button-color-disabled: var(--spectrum-disabled-content-color);
+	--spectrum-breadcrumbs-focus-indicator-color: var(--spectrum-focus-indicator-color);
 }
 
-.spectrum-Breadcrumbs--compact {
-  block-size: var(--mod-breadcrumbs-block-size-compact, var(--spectrum-breadcrumbs-block-size-compact));
+/* Variant: large */
+.spectrum-Breadcrumbs--large {
+	--spectrum-breadcrumbs-block-size: var(--spectrum-component-size-200);
+	--spectrum-breadcrumbs-font-size: var(--spectrum-font-size-200);
+	--spectrum-breadcrumbs-font-size-current: var(--spectrum-font-size-200);
+
+	--spectrum-breadcrumbs-separator-spacing-inline: var(--spectrum-breadcrumbs-text-to-separator-large);
+	--spectrum-breadcrumbs-text-spacing-block-start: var(--spectrum-component-top-to-text-200);
+	--spectrum-breadcrumbs-text-spacing-block-end: var(--spectrum-component-bottom-to-text-200);
+	--spectrum-breadcrumbs-icon-spacing-block: var(--spectrum-breadcrumbs-top-to-separator-large);
+	--spectrum-breadcrumbs-inline-start: var(--spectrum-breadcrumbs-start-edge-to-text-large);
 }
 
+/* Variant: multiline */
 .spectrum-Breadcrumbs--multiline {
-  block-size: var(--mod-breadcrumbs-block-size-multiline, var(--spectrum-breadcrumbs-block-size-multiline));
+	--spectrum-breadcrumbs-block-size: var(--spectrum-breadcrumbs-height-multiline);
+	--spectrum-breadcrumbs-font-size: var(--spectrum-font-size-75);
+	--spectrum-breadcrumbs-font-size-current: var(--spectrum-heading-size-l);
+	--spectrum-breadcrumbs-font-weight-current: var(--spectrum-extra-bold-font-weight);
+
+	--spectrum-breadcrumbs-separator-spacing-inline: var(--spectrum-breadcrumbs-text-to-separator-multiline);
+	--spectrum-breadcrumbs-text-spacing-block-start: var(--spectrum-breadcrumbs-top-to-text-multiline);
+	--spectrum-breadcrumbs-text-spacing-block-end: var(--spectrum-breadcrumbs-top-text-to-bottom-text);
+	--spectrum-breadcrumbs-icon-spacing-block: var(--spectrum-breadcrumbs-top-to-separator-multiline) var(--spectrum-breadcrumbs-separator-to-bottom-text-multiline);
+	--spectrum-breadcrumbs-title-spacing-block-start: 0;
+	--spectrum-breadcrumbs-title-spacing-block-end: var(--spectrum-breadcrumbs-bottom-to-text-multiline);
+	--spectrum-breadcrumbs-inline-start: var(--spectrum-breadcrumbs-start-edge-to-text-multiline);
+	--spectrum-breadcrumbs-action-button-spacing-block: var(--spectrum-breadcrumbs-top-to-truncated-menu) var(--spectrum-breadcrumbs-truncated-menu-to-bottom-text);
+}
 
-  flex-wrap: wrap;
-  align-content: center;
+.spectrum-Breadcrumbs {
+	list-style-type: none;
+	margin: 0;
+	padding-inline: 0 var(--mod-breadcrumbs-inline-end, var(--spectrum-breadcrumbs-inline-end));
+	block-size: var(--mod-breadcrumbs-block-size, var(--spectrum-breadcrumbs-block-size));
+
+	display: flex;
+	flex-flow: row nowrap;
+	align-items: center;
+	justify-content: flex-start;
+	flex: 1 0 0%;
```

Comment:
This isn't something introduced in the migration, but I noticed this in the inspector and I'm wondering if there's any consequence to removing it:

![Screenshot 2024-11-18 at 1 48 04 PM](https://github.com/user-attachments/assets/65720c48-8b32-4ab3-a337-3337b6c6ff94)

Reply:
Thanks. I've removed this property from the root class, as it only applies to a flex item.